catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/validation/impl/validator/EnumerationValidator.java

```diff
@@ -54,10 +53,8 @@ public EnumerationValidator(final Set<String> values, final boolean ignoreCase)
 
         this.ignoreCase = ignoreCase;
 
-        this.values = Collections.unmodifiableSet(values.stream()
-                .filter(Objects::nonNull)
-                .map(ignoreCase ? String::toLowerCase : String::toString)
-                .collect(Collectors.toSet()));
+        this.values = new TreeSet<>(ignoreCase? String.CASE_INSENSITIVE_ORDER : null);
```

Comment:
Though clearly checkstyle doesn't care. And in this case it makes it slightly more terse, so ima leave it in less someone has really strong opinions.